proposals/deliverable-proposals/profiles.md

```diff
@@ -12,11 +12,40 @@ and on resource constrained devices.
-* Digital Twin Profile (addressing behavioral, accessibility and UI requirements)
-* Cloud Profile (addressing requirements of large scale requirements)
-* Constraint Device Profile (addressing requirements of devices with a small footprint)
-* Profiles for other protocols (e.g. CoAP, MQTT)
+In addition, specification work for other profiles will be carried out. 
+Initial use cases and requirements for other profiles are described in https://www.w3.org/TR/wot-usecases .
+
+To address some of these use cases, the following profiles have been proposed:  
+
+### Digital Twin Profile
+ 
+Digital twins enable defining a model of a thing, person or process. They are used as digital proxies for counterparts in the real world to better understand situations in the real world and act on state changes. 
+
+Using a digital twin allows businesses to analyze their physical assets to troubleshoot in real time, predict future problems, minimize downtime, and perform simulations to create new business opportunities. A digital twin may also be called a twin or a shadow. 
+
+Digital twins have different roles. They could be models of individual discrete things, composite twins or organisational twins of more complex entities, such as factories or cities.
+Some use cases for digital twins have been described in the **WoT use cases and requirements** document, e.g. for Smart Cities, Production Monitoring, Health and Agriculture scenarios.
+
+The *Digital Twin Profile* enables using the thing description for defining digital twins that address these roles, use cases and requrements. 
+
+### Cloud Profile
+
+Large scale deployments, for example in production lines for industrial manufacturing consist of multiple machines, where each machine incorporates sensors for various values. A failure of a single machine can cause defective products or a stop of the entire production.
+
+Big data analysis enables to identify behavioral patterns across multiple production lines of the entire production plant and across multiple plants.
+
+The results of this analysis can be used for optimizing consumption of raw materials, checking the status of production lines and plants and predicting and preventing fault conditions.
+
+The *Cloud Profile* enables large scale deployments of devices across different manufacturers. 
+
+### Profiles for constrained devices and other protocols
+
+The current WoT Profile specification contains a protocol binding for HTTP(s). 
+In an industrial environment individual actuators and production devices use several other protocols. Examples include MQTT, OPC UA, Modbus, and others. Gathering data from these devices, e.g. to support digital twins or big data use cases requires a protocol binding and additional semantic guarantees that enable them.
+
+As part of the profile work item, other profiles may be defined, based on concrete use cases and requirements.
```

Comment:
I support the idea of a separate profile for constrained devices which don't have the resources to implement the existing HTTP profiles.

However, I think including protocol bindings for four different protocols in a single profile would be too complicated and wouldn't actually help with interoperability since devices using the different protocol bindings wouldn't be interoperable with _each other_.

I instead suggest a single "**CoAP Profile**" which uses CoAP and CBOR instead of HTTP and JSON, for constrained devices.

Reply:
> I support the idea of a separate profile for constrained devices which don't have the resources to implement the existing HTTP profiles.
> 
> However, I think including protocol bindings for four different protocols in a single profile would be too complicated and wouldn't actually help with interoperability since devices using the different protocol bindings wouldn't be interoperable with _each other_.
> 
> I instead suggest a single "**CoAP Profile**" which uses CoAP and CBOR instead of HTTP and JSON, for constrained devices.

There was a misunderstanding on that part of the proposal. It is not meant to combine protocol bindings for several protocols into a single profile. I'll improve the text.